source/Projectile.cpp

```diff
@@ -199,6 +198,15 @@ void Projectile::Move(vector<Visual> &visuals, vector<Projectile> &projectiles)
 			}
 		}
 	}
+	// If the weapon is homing, is radar-guided, has lost its lock, and the
+	// target has jamming, give a chance for the missile to turn in a random
+	// direction, in proportion to the strength of the jamming and the missile's
+	// tracking quality
+	else if(target && homing && weapon->RadarTracking() && target->Attributes().Get("radar jamming") > 0.0)
+	{
+		if(Random::Real() < (1 - (weapon->RadarTracking() * 2 / target->Attributes().Get("radar jamming"))) )
```

Comment:
I'm guessing that `Random::Real()` is a number between 0 and 1.

Does this mean that a small radar jammer (2 jamming) has a 10% chance to cause a Sidewinder to go haywire while a large radar jammer (10 jamming) has an 82% chance?

Reply:
Yes. Note that this condition is only evaluated once the missile has already lost its lock. Maybe I should also make this only happen if the missile is within the jamming radius?

Reply:
`Maybe I should also make this only happen if the missile is within the jamming radius?`

Can a missile lose its lock outside of the jamming radius?

Reply:
> `Maybe I should also make this only happen if the missile is within the jamming radius?`
> 
> Can a [radar-guided] missile lose its lock outside of the jamming radius?

Not from jamming. It can still lose its lock due to imperfect radar tracking, though.